interactive_ai/tests/e2e/features/steps/model_test_management.py

```diff
@@ -0,0 +1,59 @@
+from behave import step, when
+from geti_client import TriggerModelTestJobRequest
+
+
+@when("the user requests to test the model on the default dataset")
+def step_user_requests_model_test(context):
+    model_group = context.models_api.get_model_groups(
+        organization_id=context.organization_id, workspace_id=context.workspace_id, project_id=context.project_id
+    ).model_groups[0]
+    model_group_id = model_group.id
+    base_model_id = model_group.models[0].id
+    optimized_models = context.models_api.get_model_detail(
+        organization_id=context.organization_id,
+        workspace_id=context.workspace_id,
+        project_id=context.project_id,
+        model_group_id=model_group_id,
+        model_id=base_model_id,
+    ).optimized_models
+    for optimized_model in optimized_models:
+        if "FP16" in optimized_model.precision:
+            optimized_model_id = optimized_model.id
+            break
+    else:
+        raise ValueError("No FP16 optimized model found in the model group")
+
+    trigger_model_test = TriggerModelTestJobRequest(
+        name="Test 1", dataset_ids=[context.dataset_id], model_group_id=model_group_id, model_id=optimized_model_id
+    )
+    model_test_response = context.tests_api.trigger_model_test_job(
+        organization_id=context.organization_id,
+        workspace_id=context.workspace_id,
+        project_id=context.project_id,
+        trigger_model_test_job_request=trigger_model_test,
+    )
+    context.job_id = model_test_response.job_id
+
+
+@step("a model test report is created")
```

Comment:
```suggestion
@then("a model test report is created")
```
I suggest using the more specific decorators (given, when, then) when possible, to make it clear what's the role of the step. For example, the role of a "then" step is to make assertions on the current state.